### PR TITLE
Unite tribler utilities

### LINKED ISSUE #1: Port to python 3 since python 2 support is soon dropped
As discussed with @qstokkink in Tribler/tribler#5029, the python 2 support is dropped in January 1st, 2020 (two days at the time of writing), therefor imo (edit: and theirs "As Python 2 support will formally be dropped in two days (!), I think it is safe to port it fully to Python 3") it is safe to port the repo (currently a relatively simple matter) to python 3.

### PR DESCRIPTION
**Pull request by Solomon1732**
Port bootstrap_creator.py to python 3 and unite tribler utilities which previously existed in previous places. Closes #1 
Edit: the added scripts are [these](https://gist.github.com/qstokkink/0b6f7aed57f698f87df4dd77feb18e9f) and [these](https://gist.github.com/qstokkink/0b1119bc458931d79b0402f80cf20749)

### CODE REVIEW

File: import_extractor.py [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+})
+std_lib = sysconfig.get_python_lib(standard_lib=True)
+for top in pathlib.Path(std_lib).iterdir():
+    if top.startswith(('dist-packages', 'site-packages')):
+        continue
+    built_in_modules.add(top.split('.')[0])
+
+# We take the top-folder as input
+location = os.path.abspath(sys.argv[1])
+hint_locations = [pathlib.Path(location, 'Tribler', 'dispersy'),
```
Comment (devos50) [inline import_extractor.py]: I think that all these `hint_locations` can be removed since we are not using any of them anymore 👍

Comment (Solomon1732) [reply import_extractor.py]: Done.

**Comment by devos50** [comment]
@Solomon1732 thanks for your contribution! I will merge this PR when the unit test builder is fixed 👍

**Comment by Solomon1732** [comment]
Happy to help 😄

**Comment by Solomon1732** [comment]
Any news on this?

**Review by xoriole — APPROVED** [review]

**Pull request merged**